.github/workflows/auto-release-bugfix.yml

```diff
@@ -38,8 +34,24 @@ jobs:
               env:
                   SOURCE_BRANCH: ${{ env.SOURCE_BRANCH }}
                   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-                  RELEASE_MANAGER: ${{ env.RELEASE_MANAGER }}
               run: |
+                  # Get the shared environment variables.
+                  # Adds the RELEASE_MANAGER environment variable.
+                  source ./ci/release/RELEASE_MANAGER.env
+
+                  # Don't do a release if there are no changes on HISTORY since
+                  # the last release.  This is done by checking to see if
+                  # HISTORY has been modified between $LATEST_VERSION and
+                  # the commit this build is running on ($GITHUB_SHA).
+                  LATEST_VERSION=$(python setup.py --version | awk -F '.' '{print $1"."$2"."$3}')
+                  if git diff --name-only $LATEST_VERSION $GITHUB_SHA | grep HISTORY.rst > /dev/null
```

Comment:
I'm sure you tested this conditional and it works, but where would one go to find out what qualifies as truthy for github actions yml? Or is it the rules for the current shell that matter here?

Reply:
Yeah, it's completely determined by the rules of the current shell.  This is a bash script here, so bash truthiness rules apply.

I still have an outstanding TODO on #144 to refactor these build scripts out of the YML so that they can be more easily run independently of github actions, and I hope that'll help clarify the nature of the scripts as well.